### PR TITLE
Change grid to use `inline-block` not `float`

### PR DESCRIPTION
**Pull request by chris-pearce**
This fixes issue #2.

### CODE REVIEW

File: layout/_l-grid.scss [hunk 1]
```diff
@@ -52,33 +52,26 @@
  */
 
 // Which breakpoints do you want to use?
-$l-grid-breakpoints:                              $default-breakpoints
+$l-grid-breakpoints:    $default-breakpoints
 !default;
```
Comment (kllevin) [inline layout/_l-grid.scss]: move this up a line.

File: layout/_l-grid.scss [hunk 2]
```diff
@@ -52,33 +52,26 @@
...
+$l-grid-gutter-base:    $spacing-base !default;
+
+$l-grid-gutter-tiny:    $spacing-third !default;
 
-$l-grid-breakpoint-toggle-table-v-align-middle:   false !default;
+$l-grid-gutter-small:   $spacing-half !default;
 
-$l-grid-breakpoint-toggle-table-v-align-bottom:   false !default;
+$l-grid-gutter-large:   $spacing-base-plus-half
+!default;
```
Comment (kllevin) [inline layout/_l-grid.scss]: move this up a line?

File: style.css [hunk 3]
```diff
@@ -4877,9 +4753,6 @@ li:before {
  * N.B. this utility is dependant on the following utility:
  *
  * - Positioning.
- *
- * @demo
- * http://codepen.io/team/westfieldlabs/full/xFHfk#overlay
  */
 /**
  * Settings.
```
Comment (kllevin) [inline style.css:4758]: why are you checking in css?

File: style.css.map [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
...,QAAsB;EAAhC,aAAQ,EAAE,QAAsB;;A+CklBpC,eACgB;E/CnlBZ,WAAQ,EAAE,QAAsB;EAAhC,cAAQ,EAAE,QAAsB;;;;;A+C8lBpC,WACY;E/C/lBR,MAAQ,EAAE,IAAsB;;A+CimBpC,WACY;E/ClmBR,OAAQ,EAAE,IAAsB;;A+CqmBpC,aACc;E/CtmBV,MAAQ,EAAE,KAAsB;;A+CwmBpC,aACc;E/CzmBV,OAAQ,EAAE,KAAsB;;A+C6mBpC,YACa;E/C9mBT,UAAQ,EAAE,IAAsB;;A+CgnBpC,YACa;E/CjnBT,WAAQ,EAAE,IAAsB;;A+ConBpC,cACe;E/CrnBX,UAAQ,EAAE,KAAsB;;A+CunBpC,cACe;E/CxnBX,WAAQ,EAAE,KAAsB;;A+C4nBpC,YACa;E/C7nBT,YAAQ,EAAE,IAAsB;;A+C+nBpC,YACa;E/ChoBT,aAAQ,EAAE,IAAsB;;A+CmoBpC,cACe;E/CpoBX,YAAQ,EAAE,KAAsB;;A+CsoBpC,cACe;E/CvoBX,aAAQ,EAAE,KAAsB;;A+C2oBpC,YACa;E/C5oBT,aAAQ,EAAE,IAAsB;;A+C8oBpC,YACa;E/C/oBT,cAAQ,EAAE,IAAsB;;A+CkpBpC,cACe;E/CnpBX,aAAQ,EAAE,KAAsB;;A+CqpBpC,cACe;E/CtpBX,cAAQ,EAAE,KAAsB;;A+C0pBpC,YACa;E/C3pBT,WAAQ,EAAE,IAAsB;;A+C6pBpC,YACa;E/C9pBT,YAAQ,EAAE,IAAsB;;A+CiqBpC,cACe;E/ClqBX,WAAQ,EAAE,KAAsB;;A+CoqBpC,cACe;E/CrqBX,YAAQ,EAAE,KAAsB;;A+CyqBpC,YACa;E/C1qBT,WAAQ,EAAE,IAAsB;EAAhC,YAAQ,EAAE,IAAsB;;A+C4qBpC,YACa;E/C7qBT,YAAQ,EAAE,IAAsB;EAAhC,aAAQ,EAAE,IAAsB;;A+CgrBpC,cACe;E/CjrBX,WAAQ,EAAE,KAAsB;EAAhC,YAAQ,EAAE,KAAsB;;A+CmrBpC,cACe;E/CprBX,YAAQ,EAAE,KAAsB;EAAhC,aAAQ,EAAE,KAAsB;;A+CwrBpC,YACa;E/CzrBT,UAAQ,EAAE,IAAsB;EAAhC,aAAQ,EAAE,IAAsB;;A+C2rBpC,YACa;E/C5rB
```
Comment (kllevin) [inline style.css.map]: what does this do?

File: layout/_l-grid.scss (hunk 1 above)
Comment (chris-pearce) [inline layout/_l-grid.scss]: done

File: layout/_l-grid.scss (hunk 2 above)
Comment (chris-pearce) [inline layout/_l-grid.scss]: done

File: style.css (hunk 3 above)
Comment (chris-pearce) [inline style.css:4758]: Cause I need it for the Bower package `main` property.

File: style.css.map (hunk 4 above)
Comment (chris-pearce) [inline style.css.map]: Just a map file that maps the `.scss` to the compiled `.css` when inspecting in a browser, it's checked in for the same reason `style.css` is.

**Pull request merged**